src/styles/main.css

```diff
@@ -1,7 +1,33 @@
-.strokeme {
+@font-face {
+    font-family: pokemonFont;
+    src: url(../assets/fonts/firered.ttf);
+}
+
+body{
+    background: url('../assets/images/background.png') center;
+}
+
+.hoi {
```

Comment:
Could you rename this back to `.strokeme` or `borderText` or something more descriptive?